nautobot_ssot/utils/orm.py

```diff
@@ -73,3 +77,29 @@ def load_typed_dict(typed_dict_class: Type, db_obj: Model) -> dict:
     for field_name in get_type_hints(typed_dict_class):
         typed_dict[field_name] = orm_attribute_lookup(db_obj, field_name)
     return typed_dict
+
+
+def set_custom_relationship_association(
+    relationship: Relationship,
+    relationship_side: RelationshipSideEnum,
+    source_object: BaseModel,
+    destination_object: BaseModel,
+):
+    """Create or update a custom relationship association between two Nautobot objects."""
```

Comment:
What is a case where updating a custom relationship association would be desired?